code/game/objects/items/weapons/storage/backpack_vr.dm

```diff
@@ -1,20 +1,31 @@
 /obj/item/weapon/storage/backpack/saddlebag
-	name = "Saddlebag"
+	name = "Horse Saddlebags"
 	desc = "A saddle that holds items. Seems slightly bulky."
 	icon = 'icons/obj/storage_vr.dmi'
 	icon_override = 'icons/mob/back_vr.dmi'
 	item_state = "saddlebag"
 	icon_state = "saddlebag"
 	max_storage_space = INVENTORY_DUFFLEBAG_SPACE //Saddlebags can hold more, like dufflebags
 	slowdown = 1 //And are slower, too...Unless you're a macro, that is.
+	var/taurtype = /datum/sprite_accessory/tail/taur/horse //Acceptable taur type to be wearing this
+	var/no_message = "You aren't the appropriate taur type to wear this!"
+
 	mob_can_equip(var/mob/living/carbon/human/H, slot, disable_warning = 0)
 		if(..())
-			if(istype(H) && istype(H.tail_style, /datum/sprite_accessory/tail/taur/horse))
-				if(H.size_multiplier >= 1.5 && slowdown == 1) //Are they a macro? Is the slowdown set to 1?
+			if(istype(H) && istype(H.tail_style, taurtype))
+				if(H.size_multiplier >= RESIZE_BIG) //Are they a macro?
 					slowdown = 0
-				if(H.size_multiplier < 1.5 && slowdown == 0) //Are they a micro/not a macro? Is the slowdown set to 0?
-					slowdown = 1
+				else
+					slowdown = initial(slowdown)
 				return 1
 			else
-				H << "<span class='warning'>You need to have a horse half to wear this.</span>"
```

Comment:
That line is red because I removed it. See the 'no_message' variable, which is the message it prints when you attempt to put it on and are told no.